src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/cloud-watch-fallback.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "CloudWatch fallback"
+metaDescription: This doc guides you through shipping your telemetry data with Cloudwatch as a fail-safe option.
+freshnessValidatedDate: never
+---
+
+You can ship New Relic Lambda monitoring data in a few different ways depending on your needs. The option that provides the strongest performance of your function and the strongest fail-safe is the CloudWatch fallback option. This means that you will ship telemetry to New Relic through the agent and use the extension to enhance the telemetry collection but, if there's a problem with the extension, you will still receive logs and payloads in CloudWatch.  
```

Comment:
The option that provides the strongest performance of your function and the strongest fail-safe includes configuring CloudWatch as a fallback. This means that you will ship telemetry to New Relic via the extension and, if there's a problem with the extension, New Relic will still receive payloads via CloudWatch.

Reply:
Thanks! I changed it to this (just to get rid of via):

The option that provides the strongest performance of your function and the strongest fail-safe includes configuring CloudWatch as a fallback. This means that you will ship telemetry to New Relic through the extension and, if there's a problem with the extension, New Relic will still receive payloads through CloudWatch.

Reply:
Done!